src/tests/multihost/alltests/test_multidomain.py

```diff
@@ -359,12 +359,13 @@ def test_0017_filesldap(self, multihost, multidomain_sssd, localusers):
         """
         multidomain_sssd(domains='local_ldap')
         multihost.client[0].service_sssd('start')
-        err_str = "groupdel: cannot remove the primary group of user 'quser0'"
+        err_str1 = "groupdel: cannot remove entry 'qgroup0' from /etc/group"
+        err_str2 = "groupdel: cannot remove the primary group of user 'quser0'"
         groupdel = 'groupdel qgroup0'
         cmd = multihost.client[0].run_command(groupdel, raiseonerr=False)
         assert cmd.returncode != 0
         print(cmd.stderr_text.strip().split('\n'))
-        assert err_str in cmd.stderr_text.strip().split('\n')
+        assert err_str1 or err_str2 in cmd.stderr_text.strip().split('\n')
```

Comment:
I would be a bit afraid here that err_str1 and err_str2 are a truthy values so it might always pass.
I would rather see there:
assert err_str1 in cmd.stderr_text or err_str2 in cmd.stderr_text, "Expected error message not found."